packages/commonwealth/client/scripts/state/api/threads/createThread.ts

```diff
@@ -38,16 +38,16 @@ const createThread = async ({
     session = null,
     hash = null,
   } = await app.sessions.signThread(address, {
-    community: chainId,
+    community: communityId,
     title,
     body,
     link: url,
     topic: topic.id,
   });
 
   const response = await axios.post(`${app.serverUrl()}/threads`, {
-    author_chain: chainId,
-    community_id: chainId,
+    author_chain: communityId,
```

Comment:
It's safe to update this since `lookupAddressIsOwnedByUser.ts` supports both `author_chain` and `author_community_id`.
```suggestion
    author_community_id: communityId,
```